src/main/groovy/edu/oregonstate/mist/personsapi/mapper/JobsMapper.groovy

```diff
@@ -9,6 +9,23 @@ import java.sql.SQLException
 
 public class JobsMapper implements ResultSetMapper<JobObject> {
     public JobObject map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        String classification
+        String classificationCode = rs.getString('EMPLOYEE_CLASSIFICATION_CODE')
+
+        if (classificationCode.startsWith('C') || classificationCode.startsWith('G')) {
+            classification = 'Classified'
+        } else if (classificationCode.startsWith('T')) {
+            classification = 'Temporary'
+        } else if (classificationCode.startsWith('X')) {
+            classification = 'Student'
+        } else if (classificationCode.startsWith('U')) {
+            classification = 'Unclassified'
+        } else if (classificationCode == 'XX') {
+            classification = 'Unpaid Appt'
```

Comment:
You can never get the `classificationCode == 'XX'` case because `classificationCode.startsWith('X')` will pass first.

Reply:
Oops, nice catch, fixed.